packages/react-components/react-search/library/src/components/SearchBox/useSearchBox.tsx

```diff
@@ -75,8 +78,8 @@ export const useSearchBox_unstable = (props: SearchBoxProps, ref: React.Ref<HTML
         {
           ...rootProps,
           ref: useMergedRefs(rootProps?.ref, searchBoxRootRef),
-          onFocus: useEventCallback(mergeCallbacks(rootProps?.onFocus, onFocus)),
-          onBlur: useEventCallback(mergeCallbacks(rootProps?.onBlur, onBlur)),
+          onFocus: mergeCallbacks(rootProps?.onFocus, onFocus),
+          onBlur: mergeCallbacks(rootProps?.onBlur, onBlur),
```

Comment:
This will return a new return due `mergeCallbacks()`, but I don't anyway on how to avoid it there.